### ISSUE TITLE
Do not print package.json in the end

### ISSUE THREAD

**Issue by sapegin** [th0r/npm-upgrade#18]
I’ve never used this feature but often I want to check versions of updated packages before and after. So a short summary would be more useful in my opinion.

I think something like this would be more useful:

```
These packages will be updated:

  jss                        ^9.3.3   →    ^9.4.0
  jss-default-unit           ^8.0.0   →    ^8.0.2
  q-i                        ^1.2.0   →    ^2.0.0
  babel-jest                ^21.2.0   →   ^22.0.3

? Update packages?
❯ Yes
  No
  Show package.json
```

What do you think?

**Comment by th0r** [comment]
I think this is the way it had to be done! Released in v1.3.0.

**Comment by sapegin** [comment]
Much better now, thanks! 🍕
